api/src/operations/mail/index.ts

```diff
@@ -12,6 +13,8 @@ export type Options = {
 	subject: string;
 };
 
+const logger = useLogger();
```

Comment:
@rijkvanzanten Do you prefer to have those calls on top-level vs. inside functions where it is used?

Reply:
Either is fine for now 👍🏻